docs/core/metrics_datadog.md

```diff
@@ -0,0 +1,235 @@
+
+## Getting started
+
+???+ tip
+    All examples shared in this documentation are available within the [project repository](https://github.com/aws-powertools/powertools-lambda-python/tree/develop/examples){target="_blank" }.
+
+Datadog provider has two global settings that will be used across all metrics emitted:
+
+| Setting              | Description                                                                     | Environment variable           | Constructor parameter |
+| -------------------- | ------------------------------------------------------------------------------- | ------------------------------ | --------------------- |
+| **Metric namespace** | Logical container where all metrics will be placed e.g. `ServerlessAirline`     | `POWERTOOLS_METRICS_NAMESPACE` | `namespace`           |
+| **Flush to log**     | Use this when you want to flush metrics to be exported through Datadog Forwarder| `DD_FLUSH_TO_LOG`              | `flush_to_log`        |
+
+Experiment to use your application or main service as the metric namespace to easily group all metrics.
+
+### Install
+
+???+ note
+    If you are using Datadog Forwarder, you can skip this step step.
+
+To adhere to Lambda best practices and effectively minimize the size of your development package, we recommend using the official Datadog layers built specifically for the SDK and extension components. Below is the template that demonstrates how to configure a SAM template with this information.
+
+```yaml hl_lines="13 14 22 24" title="AWS Serverless Application Model (SAM) example"
+--8<-- "examples/metrics_datadog/sam/template.yaml"
+```
+
+If you prefer not to utilize the Datadog SDK provided through the Datadog layer, add `aws-lambda-powertools[datadog]` as a dependency in your preferred tool: _e.g._, _requirements.txt_, _pyproject.toml_. This will ensure you have the required dependencies before using Datadog provider.
+
+### Creating metrics
+
+You can create metrics using `add_metric`. Optional parameter such as timestamp can be included, but if not provided, the Datadog Provider will automatically use the current timestamp by default.
+
+???+ tip
+	You can initialize DadatadogMetrics in any other module too. It'll keep track of your aggregate metrics in memory to optimize costs (one blob instead of multiples).
+
+=== "add_metrics_with_provider.py"
+
+    ```python hl_lines="6 12"
+    --8<-- "examples/metrics_datadog/src/add_metrics_with_provider.py"
+    ```
+
+=== "add_metrics_without_provider.py"
+
+    ```python hl_lines="11"
+    --8<-- "examples/metrics_datadog/src/add_metrics_without_provider.py"
+    ```
+
+???+ warning "Warning: Do not create metrics outside the handler"
+    Metrics added in the global scope will only be added during cold start. Disregard if you that's the intended behavior.
+
+### Adding tags
+
```

Comment:
add a quick note on what Tags enable customers to do. similar link hell anti-pattern